e2e/e2e.js

```diff
@@ -170,7 +169,9 @@ process.env["CYPRESS_FRONTITY_MODE"] =
           await cypress.run({
             env: { WORDPRESS_VERSION: wpVersion },
             browser,
-            spec: `./integration/${suite}/**/*.spec.js`,
+            spec: spec
+              ? `./integration/**/**/${spec}.spec.js`
```

Comment:
This double globstar is not required.

```suggestion
              ? `./integration/**/${spec}.spec.js`
```